### PR TITLE
Allow closing of context menu even when disabled

### PR DESCRIPTION
**Pull request by pads**
This change allows the context menu to be closed when in a disabled state.  This helps fix issues whereby a context menu can be open but something external happens that triggers the condition set against the directive attribute `context-menu-disabled` to be true, thus rendering it disabled.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
+dist: xenial
+services:
+  - xvfb
```
Comment (pads) [inline .travis.yml:5]: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-to-run-tests-that-require-a-gui

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 language: node_js
 node_js:
-  - stable
+  - 10
```
Comment (pads) [inline .travis.yml:8]: Going to node 12 breaks gulp, this is the highest we can use for the time being.

File: dist/ng-context-menu.js [hunk 3]
```diff
@@ -107,16 +107,15 @@
...
                 $scope.$apply(function() {
                   close(ContextMenuService.menuElement);
                 });
               }
             }
 
             function handleClickEvent(event) {
-              if (!$scope.disabled() &&
-                opened &&
+              if (opened &&
```
Comment (steveellisdev) [inline dist/ng-context-menu.js:118]: Does this work ok in Firefox? The comment a few lines further down suggests it does something weird with the events where we get a click event and a `contextmenu` event.

Comment (pads) [reply dist/ng-context-menu.js:118]: I've tested this in Firefox 80 and it works fine.

File: protractor.conf.js [hunk 4]
```diff
@@ -1,7 +1,10 @@
 var outputDir = 'test-result';
 var config = {
   capabilities: {
-    browserName: 'firefox'
+    browserName: 'chrome',
+    chromeOptions: {
+      w3c: false
```
Comment (pads) [inline protractor.conf.js:6]: Switched to chrome with this config so that the right click action in the browser test can work.

**Pull request merged**